airflow/providers/apache/livy/example_dags/livy_example.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+This is an example DAG which uses the LivyOperator.
+The tasks below trigger the computation of pi on the Spark instance
+using the Java and Python executables provided in the example library.
+"""
+
+import airflow
+from airflow import DAG
+from airflow.providers.apache.livy.operators.livy_operator import LivyOperator
+
+args = {
+    'owner': 'airflow',
+    'email': ['airflow@example.com'],
+    'depends_on_past': False,
+    'start_date': airflow.utils.dates.days_ago(2)
+}
+
+with DAG(
+    dag_id='example_livy_operator',
```

Comment:
Should we change the filename to dag_id value?

Reply:
I don't mind either way -- what do we do elsewhere?

Reply:
seems to be the trend. Fixing